onnx_chainer/functions/normalization.py

```diff
@@ -7,23 +7,28 @@
 
 def convert_BatchNormalization(func, opset_version, input_names,
                                num_outputs, context, parameters):
-    # Add running_mean and running_var to graph
-    running_mean = chainer.Parameter(func.running_mean)
-    parameters.append(running_mean)
-    input_names.append(context.get_name(running_mean))
-
-    running_var = chainer.Parameter(func.running_var)
-    parameters.append(running_var)
-    input_names.append(context.get_name(running_var))
-
-    unique_layer_name = '{}_{}'.format(func.__class__.__name__,
-                                       context.get_name(func))
-    num_outputs += [
-        unique_layer_name + '_mean',
-        unique_layer_name + '_var',
-        unique_layer_name + '_saved_mean',
-        unique_layer_name + '_saved_var'
-    ]
+    x = func.inputs[0].get_variable().data
```

Comment:
It'd be nice if we can share the code with FixedBN, but it's up to you

Reply:
fixed, thanks for advice